Makefile

```diff
@@ -57,21 +73,17 @@ build-natives:
 	@env GOOS=windows GOARCH=$$arch CC=x86_64-w64-mingw32-gcc CXX=x86_64-w64-mingw32-g++ \
 		$(GO) build $(GO_LDFLAGS_WIN) $(BUILDV) -tags $(TAGS) nolzma -o $(PKGNAME)-windows.exe ;
 
-build-contained:
-	rm -f mender-artifact && \
-	image_id=$$(docker build -f Dockerfile . | awk '/Successfully built/{print $$NF;}') && \
-	docker run --rm --entrypoint "/bin/sh" -v $(shell pwd):/binary $$image_id -c "cp /go/bin/mender-artifact /binary" && \
-	docker image rm $$image_id
+build-contained: $(DOCKER_BUILD_FLAG_BASE)
+	docker run --rm --entrypoint "/bin/sh" -u $$UID:$(shell id -g) -v $(shell pwd):/binary $(DOCKER_IMAGE_NAME_BASE) -c "cp /go/bin/$(PKGNAME) /binary"
 
-build-natives-contained:
-	rm -f mender-artifact && \
-	image_id=$$(docker build -f Dockerfile.binaries . | awk '/Successfully built/{print $$NF;}') && \
-	docker run --rm --entrypoint "/bin/sh" -v $(shell pwd):/binary $$image_id -c "cp /go/bin/mender-artifact* /binary" && \
-	docker image rm $$image_id
+build-natives-contained: $(DOCKER_BUILD_FLAG_NATIVES)
+	docker run --rm --entrypoint "/bin/sh" -u $$UID:$(shell id -g) -v $(shell pwd):/binary $(DOCKER_IMAGE_NAME_NATIVES) -c "cp /go/bin/$(PKGNAME)* /binary"
 
 goinstall:
 	@$(GO) install $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS)
 
+install-all: install install-autocomplete-scripts
```

Comment:
This should be in a separate commit.

Reply:
I'm sorry, it should've went into 3733b77bcc8d01ed7e5be2c7f73e1131e1f84932.
Will fix it ASAP

Reply:
Force pushed the change just now.
Sorry it took so long.